src/components/tabs/hooks/use-tab-items.ts

```diff
@@ -33,40 +33,64 @@ function useTabItems({
   // Unique ID used to construct `tabId` & `panelId`
   const uniqueId = useId()
   // Raw items, to be validated & updated when children change
-  const [rawTabItem, setRawTabItems] = useState<RawTabItem[]>([])
+  const [rawTabItems, setRawTabItems] = useState<RawTabItem[]>(
```

Comment:
TLDR; - this change splits out the `children → rawTabItem` & `rawTabItem → tabItem` logic into separate functions, so that we can use those functions to get the initial set of items.